dtformats/indx_directory_entry.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+"""INDX entries """
+
+import os
+
+from dtformats import data_format
+from dtformats.errors import ParseError
+
+class INDXRecord(data_format.BinaryDataFile):
```

Comment:
Limit the use of abbreviation, for international audiences these lead to unnecessary additional confusion